_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -0,0 +1,283 @@
+---
+id: bc4a75
+name: ARIA required owned elements
+rule_type: atomic
+description: |
+ This rule checks that each role has at least one of its required owned elements.
+accessibility_requirements:
+ wcag20:1.3.1: # Info and Relationships (A)
+  forConformance: true
+  failed: not satisfied
+  passed: further testing needed
+  inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+- CSS Styling
+authors:
+- Audrey Maniez
+- Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies) and has an explicit [semantic role](#semantic-role) that has [WAI-ARIA required owned elements](https://www.w3.org/TR/wai-aria/#mustContain), except when the owned element has an implicit semantic role that is identical to its explicit semantic role.
+
+## Expectation
+
+The target element [owns](#owned-by) at least one instance of one of its [WAI-ARIA required owned element](https://www.w3.org/TR/wai-aria-1.1/#mustContain).
+
+**Note:**
+- When a widget is missing required owned elements due to script execution or loading, authors [MUST]((w3.org/TR/wai-aria-1.1/#mustContain)) mark a containing element with `aria-busy` equal to `true`.
```

Comment:
Shouldn't that note be an actual expectation? eg, "the target element either owns one of its `mustContain` or has `aria-busy=true`".

The way it’s written now, it seems that `<ul role="list" aria-busy=true></ul>` is failing the rule (no list item) while this is perfectly OK thanks to the `aria-busy` attribute.

Maybe we can also add an example with `aria-busy` to illustrate this.